Archipelago.MultiClient.Net/Helpers/LocationCheckHelper.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using Archipelago.MultiClient.Net.Cache;
 using Archipelago.MultiClient.Net.Enums;
 using Archipelago.MultiClient.Net.Packets;
+using System;

```

Comment:
This causes GH to spit out a warning about line 10. Can we deal with it somehow?